plugins/source-terasology-modules/gatsby-node.js

```diff
@@ -0,0 +1,169 @@
+const { graphql } = require("@octokit/graphql");
+const { DateTime } = require("luxon");
+
+const PLUGIN_NAME = "source-terasology-modules";
+
+const query = `
+query Modules($cursor:String) {
+  organization(login: "Terasology") {
+    repositories(first: 20, after: $cursor) {
+      nodes {
+        id
+        name
+        url
+        description
+        homepageUrl
+        openGraphImageUrl
+        usesCustomOpenGraphImage
+
+        moduleTxt: object(expression: "develop:module.txt") {
+          ...on Blob {
+            text
+          }
+        }
+
+        readme: object(expression: "develop:README.md") {
+          ...on Blob {
+            text
+          }
+        }
+      }
+      pageInfo {
+        hasNextPage
+        endCursor
+      }
+    }
+  }
+}
+`;
+
+exports.onPreInit = () => console.log("Loaded source-terasology-modules");
+
+exports.sourceNodes = async (
+  {
+    actions: { createNode },
+    createContentDigest,
+    createNodeId,
+    reporter,
+    cache,
+  },
+  { accessToken }
+) => {
+  const gql = graphql.defaults({
+    headers: {
+      authorization: `token ${accessToken}`,
+    },
+  });
+
+  const lastFetchedKey = "terasology-modules-last-fetched";
+  const dataKey = "terasology-modules-data";
+
+  const now = DateTime.utc();
+  const lastFetched = DateTime.fromISO(await cache.get(lastFetchedKey), {
+    zone: "utc",
+  });
+
+  let repositories = [];
+
+  if (lastFetched.plus({ hours: 12 }) > now) {
```

Comment:
I would argue we can even let this be 24 hours.
```suggestion
  if (lastFetched.plus({ hours: 24 }) > now) {
```
or can we use days here?
```suggestion
  if (lastFetched.plus({ days: 1 }) > now) {
```